src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt

```diff
@@ -92,10 +116,10 @@ class OnOffClientFragment : Fragment() {
     }
 
     commandStatusTv.text =
-        requireContext().getString(R.string.send_command_type_label_text, chipCommandType.name)
+        requireContext().getString(R.string.send_command_type_label_text, chipCommandType.name, levelValue)
 
     try {
-      deviceController.beginSendCommand(commandType)
+      deviceController.beginSendCommand(commandType, ( 0xff and (levelValue ?: 0)))
```

Comment:
Please consider adding a comment explaining the level value conversion.

Reply:
Comment added.